fdbserver/workloads/FuzzApiCorrectness.actor.cpp

```diff
@@ -242,7 +242,6 @@ struct FuzzApiCorrectnessWorkload : TestWorkload {
 		std::vector<Future<Void>> tenantFutures;
 		for (int i = 0; i < self->numTenants + 1; ++i) {
 			TenantName tenantName = getTenant(i);
-			self->tenants.push_back(self->db->openTenant(tenantName));
```

Comment:
Now that this loop isn't opening the tenants, we can update the loop condition and remove the `if` inside of here